routers/api/v1/repo/repo.go

```diff
@@ -1095,3 +1095,17 @@ func GetIssueTemplates(ctx *context.APIContext) {
 
 	ctx.JSON(http.StatusOK, ctx.IssueTemplatesFromDefaultBranch())
 }
+
+// GetIssueConfig returns the issue config for a repo
+func GetIssueConfig(ctx *context.APIContext) {
+	// swagger:operation GET /repos/{owner}/{repo}/issue_config repository repoGetIssueConfig
+	// ---
+	// summary: Returns the issue config for a repo
+	// produces:
+	// - application/json
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/IssueConfig"
```

Comment:
Erm… Given that you now offer an API route, I don't think we can afford not to convert it to `string`, otherwise we **will** get backwards compatibility problems.
Also, we could think about adding a query param that controls which branch to query.

Reply:
I'm still thinking that converting to string is a bad idea. Adding a second boolean setting is much better IMHO.

Reply:
> ... we will get backwards compatibility problems.

I don't get it, like what? 

> ... adding a query param that controls which branch to query.

It is not correct, the `IssueConfig` only exists in the default branch, if you add a `config.yaml` in another branch, it's just a file and won't work, only the `config.yaml` in default branch is `IssueConfig`.